src/System.IO.MemoryMappedFiles/src/System.IO.MemoryMappedFiles.csproj

```diff
@@ -13,12 +13,15 @@
     <ResourcesSourceOutputDirectory Condition="'$(TargetGroup)' == 'net46'">None</ResourcesSourceOutputDirectory>
     <PackageTargetFramework Condition="'$(TargetGroup)' == ''">netstandard1.3</PackageTargetFramework>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.3</NuGetTargetMoniker>
+    <DefineConstants Condition="'$(TargetGroup)' == 'netcore50'">NETCORE50</DefineConstants>
```

Comment:
Woops, forgot to delete that.  I first implemented this using if-defs before splitting it out by files.